string movement_type
uint32 duration
---
string movement_type